source/ContentScript/index.ts

```diff
@@ -67,14 +71,18 @@ function reportAllStorage(): void {
 }
 
 function reportPageUnloaded(): void {
-  Browser.runtime.sendMessage({
-    type: 'reportEvent',
-    data: new ReportedEvent('pageUnload').toString(),
+  Browser.storage.sync.get({zapenable: true}).then((items) => {
```

Comment:
It would be great if this could be extracted into a function rather than repeat in several places.